presentation/build.gradle

```diff
@@ -50,6 +50,11 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             signingConfig signingConfigs.release
         }
+        debug {
+            applicationIdSuffix '.debug'
+            debuggable true
+            minifyEnabled false
+        }
```

Comment:
This code adds a test version of same app with name DQKSMS so that I can test on phone along with prod still installed.

We can get rid of this if you feel its not required after testing